### DISCUSSION TITLE
How we can create a custom button?

### DISCUSSION THREAD

**Discussion by dvgmdvgm** [vfsfitvnm/frida-il2cpp-bridge#110] [Q&A]
Hello.
Is it possible to make some custom button via default Unity GUI? With custom size, text, position, etc.
Like some custom mod menu

**Comment by vfsfitvnm** [comment]
@dvgmdvgm 
I never used Unity so I don't know, however keep in mind Il2Cpp compiles C# ahead of time, so it won't be possible to create classes at runtime (just saying, I don't know if this is a requirement).

You can use Frida's Java API (on Android) to create it (I'm 100% sure you can), but I guess the least painful solution is to build an application on your own and then communicate with the target process via RPC (e.g. android broadcasts, sockets).
